### ISSUE TITLE
PHP 7.3 incompatibility

### ISSUE THREAD

**Issue by kevinlisota** [GreenTreeLabs/post-snippets#97]
Just updated our site to PHP 7.3.1. Our debug log is filling with this error, generated by Post Snippets
```
PHP Notice:  compact(): Undefined variable:  in /wp-content/plugins/post-snippets/src/PostSnippets/Shortcode.php on line 38
```
Any pointers on how to fix?

**Comment by archon810** [comment]
Same issue here now that we updated to PHP 7.3.

@davdebcom Any chance you can take a look and issue a patch?

Thanks!

**Comment by archon810** [comment]
Cross-linking the issue on WP's support forum https://wordpress.org/support/topic/error-message-undefined-variable.

**Comment by GreenTreeLabs** [comment]
Fixed with 3.0.10
